### DISCUSSION TITLE
GraphQL type for JSON object

### DISCUSSION THREAD

**Discussion by japrozs** [MichalLytek/type-graphql#1064] [Q&A]
Hi all,
I just wanted to know how to specify the type of a JSON object in type-graphql. I'm using `passport` OAuth in a project and wish to store the raw data in my database. The raw data is in a JSON object but I can't seem to find the TypeGraphQL type for a JSON object. 

Please help

**Comment by MichalLytek** [comment]
Use GraphQL JSON scalar:
https://github.com/taion/graphql-type-json
